### DISCUSSION TITLE
Flint 2 Router (Adguard Home) - DNS Proxy i/o timeouts

### DISCUSSION THREAD

**Discussion by houmi** [AdguardTeam/AdGuardHome#7377] [Q&A]
I am seeing a lot of these issues on a brand new Flint 2 (GL-MT6000) Router and wondered if anyone has any ideas what might cause these issues.

> Mon Oct 21 05:16:25 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:16:25.779165 [error] dnsproxy: upstream [::1]:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001748177s: exchanging with [::1]:53 over udp: read udp [::1]:57729->[::1]:53: i/o timeout
Mon Oct 21 05:26:15 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:26:15.448642 [error] dnsproxy: 127.0.0.1:53: response received over udp: "exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:43589->127.0.0.1:53: i/o timeout"
Mon Oct 21 05:26:15 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:26:15.448746 [error] dnsproxy: upstream 127.0.0.1:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001387993s: exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:43589->127.0.0.1:53: i/o timeout
Mon Oct 21 05:26:15 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:26:15.449600 [error] dnsproxy: [::1]:53: response received over udp: "exchanging with [::1]:53 over udp: read udp [::1]:42494->[::1]:53: i/o timeout"
Mon Oct 21 05:26:15 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:26:15.449691 [error] dnsproxy: upstream [::1]:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001030382s: exchanging with [::1]:53 over udp: read udp [::1]:42494->[::1]:53: i/o timeout
Mon Oct 21 05:27:22 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:27:22.589295 [error] dnsproxy: [::1]:53: response received over udp: "exchanging with [::1]:53 over udp: read udp [::1]:54949->[::1]:53: i/o timeout"
Mon Oct 21 05:27:22 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:27:22.589297 [error] dnsproxy: 127.0.0.1:53: response received over udp: "exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:35207->127.0.0.1:53: i/o timeout"
Mon Oct 21 05:27:22 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:27:22.589414 [error] dnsproxy: upstream [::1]:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.002332973s: exchanging with [::1]:53 over udp: read udp [::1]:54949->[::1]:53: i/o timeout
Mon Oct 21 05:27:22 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:27:22.589417 [error] dnsproxy: upstream 127.0.0.1:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001099504s: exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:35207->127.0.0.1:53: i/o timeout
Mon Oct 21 05:30:49 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:30:49.767231 [error] dnsproxy: 127.0.0.1:53: response received over udp: "exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:54048->127.0.0.1:53: i/o timeout"
Mon Oct 21 05:30:49 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:30:49.767338 [error] dnsproxy: upstream 127.0.0.1:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001838441s: exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:54048->127.0.0.1:53: i/o timeout
Mon Oct 21 05:30:49 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:30:49.767264 [error] dnsproxy: 127.0.0.1:53: response received over udp: "exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:34430->127.0.0.1:53: i/o timeout"
Mon Oct 21 05:30:49 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:30:49.767422 [error] dnsproxy: upstream 127.0.0.1:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.000870178s: exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:34430->127.0.0.1:53: i/o timeout
Mon Oct 21 05:31:30 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:31:30.744919 [error] dnsproxy: [::1]:53: response received over udp: "exchanging with [::1]:53 over udp: read udp [::1]:57582->[::1]:53: i/o timeout"
Mon Oct 21 05:31:30 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:31:30.745091 [error] dnsproxy: upstream [::1]:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.001915354s: exchanging with [::1]:53 over udp: read udp [::1]:57582->[::1]:53: i/o timeout
Mon Oct 21 05:31:30 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:31:30.744920 [error] dnsproxy: 127.0.0.1:53: response received over udp: "exchanging with 127.0.0.1:53 over udp: read udp 127.0.0.1:56689->127.0.0.1:53: i/o timeout"
Mon Oct 21 05:31:30 2024 user.notice AdGuardHome[8307]: 2024/10/21 05:31:30.745174 [error] dnsproxy: upstream 127.0.0.1:53 failed to exchange ;lb._dns-sd._udp.0.8.168.192.in-addr.arpa.	IN	 PTR in 2.000999377s: exchanging with 127.0.0.1:53

**Comment by juanluisbaptiste** [comment]
I have the same issue with a new Flint 2 router, I have tried ALL the DNS providers [listed on AGH documentation](https://adguard-dns.io/kb/general/dns-providers/) and all timeout. AGH even timeouts with local DNS names (ex. nas.lan) configured in the DNS rewrites page of AGH settings.
